Mathlib/RingTheory/IntegralClosure/IsIntegralClosure/Basic.lean

```diff
@@ -356,6 +356,33 @@ theorem isIntegral_leadingCoeff_smul [Algebra R S] (h : aeval x p = 0) :
 
 end
 
+lemma Polynomial.Monic.quotient_isIntegralElem_of_monic {g : S[X]} (mon : g.Monic) {I : Ideal S[X]}
```

Comment:
The `_of_monic` at the end of the name might be redundant. I would suggest renaming it to `Polynomial.Monic.quotient_isIntegralElem`.